src/som/vm/constants/KernelObj.java

```diff
@@ -33,6 +33,17 @@ public static Object signalException(final String selector, final Object receive
     return disp.invoke(new Object[] {KernelObj.kernel, rcvr.getSOMClass()});
   }
 
+  public static Object signalException(final String selector, final String receiver) {
```

Comment:
Hm, I don't like the code duplication with the method above, and the name doesn't isn't really telling me why there are two methods. They do slightly different things, but perhaps the first version is kind of wrong.

Also, would be good to clean up all the duplication around exception throwing. There are quite a number of uses of lookupPrivate, which indicate the same pattern (PathPrims, ArraySetAllStrategy, ...)